### DISCUSSION TITLE
Unable to make a Bandcamp fix work using site fixes in Developer Tools

### DISCUSSION THREAD

**Discussion by cremasterjedi** [darkreader/darkreader#13368] [Q&A]
Hello!  Dark Reader used to avoid styling background images on bands' individual sites on Bandcamp.com - it seems to now cover them up with the DR base background color.  I've tried to use CSS in the Developer Tools to force transparency:

CSS
html {
    background-color: #0000 !important;
}

but I can't get it to stick - the extension seems to be overriding it.  I can make the adjustment directly in the browser's DevTools so I know I'm on the right track.  Can anyone help me figure this out?  Thanks!

**Comment by Myshor** [comment]
If this is still needed to help - can you share the URL and screen what exactly you don't like and try to fix?
